tools/buildsteps/windows/BuildSetup.bat

```diff
@@ -41,10 +41,26 @@ FOR %%b in (%*) DO (
   IF %%b==sh SET useshell=sh
 )
 
-SET PreferredToolArchitecture=x64
-SET buildconfig=Release
-set WORKSPACE=%base_dir%\kodi-build.%TARGET_PLATFORM%
+IF DEFINED PreferredToolArchitecture (
+  echo "Build Tool Arch: %PreferredToolArchitecture%"
+) else (
+  echo "Setting Default Tool Arch: %PROCESSOR_ARCHITECTURE%"
+  SET PreferredToolArchitecture=%PROCESSOR_ARCHITECTURE%
+)
+
+IF DEFINED buildconfig (
+  echo "Build Configuration: %buildconfig%"
```

Comment:
```suggestion
  echo Build Configuration: %buildconfig%
```

With ECHO don't need to use quotes, otherwise they are displayed as part of the string.
In this and others.